Frontend/ui-driver/src/Screens/BookingOptionsScreen/Handler.purs

```diff
@@ -15,7 +15,7 @@ bookingOptions = do
   (GlobalState state) <- getState
   action <- lift $ lift $ runScreen $ BookingOptionsScreen.screen state.bookingOptionsScreen
   case action of
-    SelectCab updatedState -> do
+    SelectCab updatedState toggleDowngrade -> do
```

Comment:
rename toggleDowngrade to downgrade

Reply:
`toggleDowngrade` is set for downgrading using toggle